### ISSUE TITLE
Error building TimeZones

### ISSUE THREAD

**Issue by paschermayr** [JuliaTime/TimeZones.jl#267]
Hi everyone, 

I am trying to build this module as another package is dependent on it, but did not succeed in building it. Does anyone know why / Is there an alternative way to build this package than the MWE below?

BR, 
Patrick

```
import Pkg
Pkg.add("TimeZones")
Pkg.build("TimeZones") #Failed
```

Error message in terminal:

```
TimeZones → `xxxxx\.julia\packages\TimeZones\OjMoF\deps\build.log`
┌ Error: Error building `TimeZones`:
│ [ Info: Downloading 2020a tzdata
│ ┌ Error: Download failed: curl: (35) schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was unable to check revocation for the certificate.  
│ └ @ Base download.jl:43
│ ERROR: LoadError: failed process: Process(`'C:\WINDOWS\System32\curl.exe' -s -S -g -L -f -o 'xxxxx\.julia\packages\TimeZones\OjMoF\deps\tzarchive\tzdata2020a.tar.gz' https://data.iana.org/time-zones/releases/tzdata2020a.tar.gz`, ProcessExited(35)) [35]
│ 
│ Stacktrace:
│  [1] pipeline_error at .\process.jl:525 [inlined]
│  [2] download_curl(::String, ::String, ::String) at .\download.jl:44
│  [3] download at .\download.jl:62 [inlined]
│  [4] tzdata_download(::String, ::String) at xxxxx\.julia\packages\TimeZones\OjMoF\src\tzdata\download.jl:89
│  [5] build(::String, ::Array{String,1}, ::String, ::String, ::String; verbose::Bool) at xxxxx\.julia\packages\TimeZones\OjMoF\src\tzdata\build.jl:46
│  [6] build(::String) at xxxxx\.julia\packages\TimeZones\OjMoF\src\tzdata\build.jl:82
│  [7] build(::String; force::Bool) at xxxxx\.julia\packages\TimeZones\OjMoF\src\build.jl:11
│  [8] build at xxxxx\.julia\packages\TimeZones\OjMoF\src\build.jl:11 [inlined] (repeats 2 times)
│  [9] top-level scope at xxxxx\.julia\packages\TimeZones\OjMoF\deps\build.jl:3
│  [10] include(::String) at .\client.jl:439
│  [11] top-level scope at none:5
│ in expression starting at xxxxx\.julia\packages\TimeZones\OjMoF\deps\build.jl:3
└ @ Pkg.Operations D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.4\Pkg\src\Operations.jl:892
```

Alternatively, `TimeZones.build()` yields the error message:
```
Error: Download failed: curl: (35) schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was unable to check revocation for the certificate.    
└ @ Base download.jl:43
```

**Comment by racinmat** [comment]
Are you able to run the curl outside of Julia without problems or not?

**Comment by omus** [comment]
In PR https://github.com/JuliaTime/TimeZones.jl/pull/441 the `deps/build.jl` was removed so this is no longer an issue. Included in TimeZones release 1.12
